packages/hop-node/src/db/GasPricesDb.ts

```diff
@@ -0,0 +1,36 @@
+import BaseDb from './BaseDb'
+import nearest from 'nearest-date'
+import { BigNumber } from 'ethers'
+import { DateTime } from 'luxon'
+import { normalizeDbItem } from './utils'
+
+export type GasPrice = {
+  chain?: string
+  timestamp?: number
+  gasPrice?: BigNumber
+}
+
+class GasPricesDb extends BaseDb {
+  async update (key: string, data: Partial<GasPrice>) {
+    return super.update(key, data)
+  }
+
+  async getNearest (chain: string, timestampSeconds: number): Promise<GasPrice> {
```

Comment:
`chain` is unused. I would expect to filter by chain here.

Reply:
you're correct, it should filter here